doc/config-lnd-remote.md

```diff
@@ -1,7 +1,46 @@
-# Configuring LiT with remote lnd node
+# Connecting LiT to a standalone LND process
+
+By default LiT assumes that `lnd` is running as a standalone process locally. However `litd` can connect to `lnd` running on a remote host.
+
+## Quickstart
+To connect Lightning Terminal to a remote LND instance first make sure your `lnd.conf` file contains the following additional configuration settings:
+```
+tlsextraip=<externally-reachable-ip-address>
+rpclisten=0.0.0.0:10009
+```
+
+Copy the following files that are located in your `~/.lnd/data/chain/bitcoin/mainnet` directory on your remote machine to `/some/folder/with/lnd/data/` on your local machine (where you’ll be running LiT):
+- tls.cert
+- admin.macaroon
+- chainnotifier.macaroon
+- invoices.macaroon
+- readonly.macaroon
+- router.macaroon
+- signer.macaroon
+- walletkit.macaroon
+
+Create a `lit.conf` file. The default location LiT will look for the configuration file depends on your operating system:
+- MacOS: `~/Library/Application Support/Lit/lit.conf`
+- Linux: `~/.lit/lit.conf`
+- Windows: `~/AppData/Roaming/Lit/lit.conf`
+
+Alternatively you can specify a different location by passing `--lit-dir=~/.lit`.  After creating `lit.conf` populate it with the following configuration settings:
+```
+remote.lnd.rpcserver=<externally-reachable-ip-address>:10009
+remote.lnd.macaroonpath=/some/folder/with/lnd/data/admin.macaroon
+remote.lnd.tlscertpath=/some/folder/with/lnd/data/tls.cert
+```
+
+Run LiT:
+```
+./litd --uipassword=UP48lm4VjqxmOxB9X9stry6VTKBRQI
+```
+
+Visit https://localhost:8443 to access LiT.
+
+
+## Configuring LiT with remote lnd node
```

Comment:
Should we change this to "Additional Configuration" or something like that? This subtitle seems redundant with the title of the doc above.

Reply:
```suggestion
## Additional Configuration
```